ubuntu-22.04-jammy-amd64-valgrind/Dockerfile-dbg

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu-22.04-jammy-amd64-valgrind
+
+USER root
+RUN apt-get -qq update && apt-get install -y ubuntu-dbgsym-keyring
+
+RUN echo "deb http://ddebs.ubuntu.com jammy main restricted universe multiverse" > /etc/apt/sources.list.d/ddebs.list
+RUN echo "deb http://ddebs.ubuntu.com jammy-updates main restricted universe multiverse" >> /etc/apt/sources.list.d/ddebs.list
+RUN perl -pi -e 's/# deb-src/deb-src/;' /etc/apt/sources.list
+
+RUN apt-get -qq update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    libfreetype6-dbgsym \
+    libtiff5-dbgsym \
+    libwebpmux3-dbgsym \
+    libwebp7-dbgsym \
+    libwebpdemux2-dbgsym \
+    libwebpmux3-dbgsym \
+    libopenjp2-7-dbgsym \
+    libharfbuzz0b-dbgsym \
+    libfribidi0-dbgsym \
+    emacs-nox
```

Comment:
We have some cache cleanup in the others to keep the layers/images smaller (re: https://github.com/python-pillow/docker-images/pull/91):

https://github.com/python-pillow/docker-images/blob/4591bf5938bca10ea280c8c1d907bd62f70efc47/ubuntu-22.04-jammy-amd64-valgrind/Dockerfile#L34

Reply:
I'm not as worried about size on this one, and there's much more of a possibility to need to inline add things to the image while working.